tests/filesystems/test_btrfs.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import Mock
+
+import pytest
+from dissect.btrfs.btrfs import INode
+from dissect.btrfs.c_btrfs import c_btrfs
+
+from dissect.target.filesystems.btrfs import BtrfsFilesystemEntry
+
+
+@pytest.mark.parametrize(
+    "sector_size, filesize, expected_blocks",
+    [
+        (0x1000, 0x343, 0x1),
+        (0x1000, 0x1000, 0x1),
+        (0x1000, 0x1001, 0x2),
+    ],
+    ids=["lower", "equal", "greater"],
+)
+def test_stat_information_file_blocksize(sector_size: int, filesize: int, expected_blocks: int) -> None:
+    entry = INode(Mock(), None)
+    entry.btrfs = Mock(sector_size=sector_size)
+    timestamp = c_btrfs.btrfs_timespec()
+    entry.inode = c_btrfs.btrfs_inode_item(
+        size=filesize, atime=timestamp, ctime=timestamp, otime=timestamp, mtime=timestamp
+    )
+
+    fs_entry = BtrfsFilesystemEntry(Mock(alt_seperator="/"), "unknown_path", entry)
```

Comment:
```suggestion
    fs_entry = BtrfsFilesystemEntry(Mock(), "some/path", entry)
```
There was a typo anyway (`alt_separator`) and it didn't error so I assume it's not needed anyway.

Reply:
It's not needed. It is currently only set in case of windows like filesystems